Examples/Swift/CustomViewController.swift

```diff
@@ -113,7 +112,7 @@ class CustomViewController: UIViewController, MGLMapViewDelegate, AVSpeechSynthe
                 self.arrowView.text = "⬆️"
             }
         }
-        self.instructionLabel.text = visualInstructionFormatter.string(leg: routeProgress.currentLeg, step: routeProgress.currentLegProgress.upComingStep)
+        self.instructionLabel.text = routeProgress.currentLegProgress.currentStepProgress.step.instructionsDisplayedAlongStep?.first?.primaryText
```

Comment:
VisualInstruction is undocumented – should we expect a developer to have any idea how to use it in a custom UI?

Reply:
Conflicted on this. I'm sort of thinking `VisualInstruction` should be documented and open to use for developers.